configure

```diff
@@ -1099,20 +1101,23 @@ do
     fi
 
     case $i in
-        arm-linux-androideabi)
-
-            if [ ! -f $CFG_ANDROID_CROSS_PATH/bin/arm-linux-androideabi-gcc ]
-            then
-                err "NDK $CFG_ANDROID_CROSS_PATH/bin/arm-linux-androideabi-gcc not found"
-            fi
-            if [ ! -f $CFG_ANDROID_CROSS_PATH/bin/arm-linux-androideabi-g++ ]
-            then
-                err "NDK $CFG_ANDROID_CROSS_PATH/bin/arm-linux-androideabi-g++ not found"
-            fi
-            if [ ! -f $CFG_ANDROID_CROSS_PATH/bin/arm-linux-androideabi-ar ]
+        *android*)
+            upperSnakeTarget=$(echo $i | tr '[:lower:]' '[:upper:]' | tr '\-' '\_')
```

Comment:
Stylistically our configure script tends to use underscores instead of camel case, e.g. `upper_snake_target`